mmv1/products/redis/Cluster.yaml

```diff
@@ -250,6 +250,55 @@ properties:
     description: |
       System assigned, unique identifier for the cluster.
     output: true
+  - name: 'automatedBackupConfig'
+    type: NestedObject
+    default_from_api: true
+    description: "The automated backup config for a instance."
+    properties:
+      - name: 'automatedBackupMode'
+        type: Enum
+        description: |
+          "Optional. The automated backup mode. If the mode is disabled, the other fields will be ignored."
+        default_value: "DISABLED"
+        enum_values:
+          - 'AUTOMATED_BACKUP_MODE_UNSPECIFIED'
+          - 'DISABLED'
+          - 'ENABLED'
+      - name: 'fixedFrequencySchedule'
+        type: NestedObject
+        description: "Optional. Trigger automated backups at a fixed frequency."
+        properties:
+          - name: 'startTime'
+            type: NestedObject
+            description: |
+              Required. The start time of every automated backup in UTC.
+              It must be set to the start of an hour. This field is required.
+            required: true
+            properties:
+              - name: 'hours'
+                type: Integer
+                description: |
+                  Hours of a day in 24 hour format. Must be greater than or equal to 0 and typically must be less than or equal to 23.
+                  An API may choose to allow the value "24:00:00" for scenarios like business closing time.
+              - name: 'minutes'
+                type: Integer
+                description: |
+                  Minutes of an hour. Must be greater than or equal to 0 and less than or equal to 59.
+              - name: 'seconds'
+                type: Integer
+                description: |
+                  Seconds of a minute. Must be greater than or equal to 0 and typically must be less than or equal to 59.
+                  An API may allow the value 60 if it allows leap-seconds.
+              - name: 'nanos'
+                type: Integer
+                description: |
+                  Fractions of seconds, in nanoseconds. Must be greater than or equal to 0 and less than or equal to 999,999,999.
+      - name: 'retention'
```

Comment:
Can you include a test configuration that omits `retention`?

Reply:
@rileykarson You would like me to remove it form the existing test or make an additional test without `retention` ?

Reply:
Based on the test failure, we should set the default value of `"3024000s"` (and then add a non-default `retention` value back to the current test)